js/build/shortcode-ui.js

```diff
@@ -740,6 +740,24 @@ var editAttributeFieldAttachment = sui.views.editAttributeField.extend( {
 	_openMediaFrame: function(e) {
 		e.preventDefault();
 		this.frame.open();
+		if ( this.model.get( 'value' ) ) {
+			var query_args = {
+				'post__in': [ this.model.get( 'value' ) ],
+				'orderby': 'post__in',
+				'perPage': 1
+			};
+			var attachments = wp.media.query( query_args );
```

Comment:
@goldenapples This is producing 0 attachments:

![image](https://cloud.githubusercontent.com/assets/36432/9252498/7e3afc6c-418d-11e5-9655-d0095af4416f.png)

Any idea why?